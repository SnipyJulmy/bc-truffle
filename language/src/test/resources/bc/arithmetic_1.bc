3.4 + 7.3
a = 3.1415926535897932384
b = a
a
b
b / 1
b
a
b
halt
