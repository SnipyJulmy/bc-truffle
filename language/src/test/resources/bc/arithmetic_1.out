10.7
3.1415926535897932384
3.1415926535897932384
3
3.1415926535897932384
3.1415926535897932384
3.1415926535897932384
